README.md

```diff
@@ -3,74 +3,204 @@
 
 An Ansible role for managing High Availability Clustering.
 
-## Requirements
+## Limitations
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+* Supported OS: RHEL 8.3+, Fedora 31+
+* The role expects High-Availability Add-on repositories to be enabled on
+  systems running RHEL.
+* The role replaces the configuration of HA Cluster on specified nodes. Any
+  settings not specified in the role variables will be lost.
+* For now, the role is only capable of configuring a basic corosync cluster.
 
 ## Role Variables
 
-A description of all input variables (i.e. variables that are defined in
-`defaults/main.yml`) for the role should go here as these form an API of the
-role.
+### Defined in `defaults/main.yml`
 
-Variables that are not intended as input, like variables defined in
-`vars/main.yml`, variables that are read from other roles and/or the global
-scope (ie. hostvars, group vars, etc.) can be also mentioned here but keep in
-mind that as these are probably not part of the role API they may change during
-the lifetime.
+#### `ha_cluster_cluster_present`
 
-Example of setting the variables:
+boolean, default: `yes`
 
+If set to `yes`, HA cluster will be configured on the hosts according to other
+variables. If set to `no`, all HA Cluster configuration will be purged from
+target hosts.
+
+#### `ha_cluster_start_on_boot`
+
+boolean, default: `yes`
+
+If set to `yes`, cluster services will be configured to start on boot. If set
+to `no`, cluster services will be configured not to start on boot.
+
+#### `ha_cluster_fence_agent_packages`
+
+list of fence agent packages to install, default: fence-agents-all, fence-virt
+
+#### `ha_cluster_hacluster_password`
+
+string, no default - must be specified
+
+Password of the `hacluster` user. This user has full access to a cluster.
+
+#### `ha_cluster_corosync_key_src`
+
+path to corosync authkey file, default: `null`
+
+Authentication and encryption key for Corosync communication. It is highly
+recommended to have a unique value for each cluster. The key should be 256
+bytes of random data.
+
+If no key is specified, a key already present on the nodes will be used. If
+nodes don't have the same key, a key from one node will be distributed to other
+nodes so that all nodes have the same key. If no node has a key, a new key will
+be generated and distributed to the nodes.
+
+If this variable is set, `ha_cluster_regenerate_keys` is ignored for this key.
```

Comment:
add text like "If value is provided, recommend vault encrypting the value. See https://docs.ansible.com/ansible/latest/user_guide/vault.htm"

Reply:
fixed in 1c2a9bf